x-pack/plugins/endpoint/public/applications/endpoint/store/hosts/reducer.ts

```diff
@@ -21,7 +20,7 @@ const initialState = (): HostListState => {
   };
 };
 
-export const hostListReducer: Reducer<HostListState, AppAction> = (
+export const hostListReducer: ImmutableReducer<HostListState, AppAction> = (
```

Comment:
another case where its probably not necessary, but it is helpful for debugging and stuff